synthea-hiv/generator/make_modules/make_hiv_module.py

```diff
@@ -0,0 +1,147 @@
+"""Creates Synthea Submodules from CSV file."""
+
+import json
+from typing import Dict, List
+
+import numpy as np
+import pandas as pd
+
+
+class CodeDisplay:
+  """Holds name and code value."""
+
+  def __init__(self, name: str, code: str):
+    self.name = name
+    self.code = code
+
+  def __str__(self):
+    return self.name
+
+  def __repr__(self):
+    return self.__str__()
+
+
+class Question(CodeDisplay):
```

Comment:
Do we need this and the next class? Why not directly use `CodeDisplay` and clarify question/answer in variable name only?